_includes/page_content.html

```diff
@@ -0,0 +1,3 @@
+<div markdown="1">
```

Comment:
Avoid marking a div as markdown if the child has HTML, it can do weird things sometimes.